templates/core/terraform/bastion.tf

```diff
@@ -1,6 +1,6 @@
 resource "azurerm_public_ip" "bastion" {
   name                = "pip-bas-${var.tre_id}"
-  resource_group_name = var.resource_group_name
+  resource_group_name = azurerm_resource_group.core.name
   location            = var.location
```

Comment:
Shall we do location whilst we are here too? (Take it from the azurerm_resource_group.core.location)

Reply:
Absolutely LOVE the "moved" - today I learned something new!